motion/motion.go

```diff
@@ -105,7 +103,7 @@ func (d *motionDetector) Detect(frame *lepton3.Frame) bool {
 	d.debug.update("delta", deltaCount)
 
 	if d.debug != nil && d.count%(debugLogSecs*lepton3.FramesHz) == 0 {
-		log.Print("motion:: " + d.debug.string("detect:n temp:all ftemp:all diff:max delta:max ffc:n"))
+		log.Print("motion:: " + d.debug.string("thresh:all detect:n temp:all ftemp:all diff:max delta:max ffc:n"))
```

Comment:
why did this get changed?

Reply:
To print out the temperature threshold changes

Reply:
Sorry, I misread the line!